cypress/page/shopping-cart.page.ts

```diff
@@ -3,7 +3,8 @@ class ShoppingCartPage {
     private goToCheckoutBtn: string;
 
     constructor() {
-        this.goToCartBtn = "[style*='display: block;'] .button-container > a";
+
+        this.goToCartBtn = ".button-container > .button-medium";
```

Comment:
this should be part of the `product-list.page.ts` file, since that page object should have two methods:

- one for the click to add to cart
- other for the click in the proceed to checkout button

then the shopping-cart.page.ts, should contain just one method, that is for the click in the proceed to checkout button.